lang/cs/Org.Apache.REEF.Network/Group/Operators/IBroadcastReceiver.cs

```diff
@@ -26,7 +28,8 @@ public interface IBroadcastReceiver<T> : IGroupCommOperator<T>
         /// <summary>
         /// Receive a message from parent BroadcastSender.
         /// </summary>
+        /// <param name="cancellationSource">The cancellation token for the data reading operation cancellation</param>
         /// <returns>The incoming message</returns>
-        T Receive();
+        T Receive(CancellationTokenSource cancellationSource = null);
```

Comment:
Do we need to mart it unstable, create a JIRA that null would be removed and then mention it as ToDo?